examples/FMP_simple_temp.jl

```diff
@@ -0,0 +1,252 @@
+# # Force Based Motion Planning
+# ![](circle_swap_pretty.gif) 
+
+# This is an example implementation of the [Force Based Motion Planning (FMP)](https://arxiv.org/pdf/1909.05415.pdf) algorithm. The algorithm is a decentralized motion
+# planning algorithm where individual agents experience attractive forces towards
+# a goal position and repulsive forces from other agents or objects in the environment. 
+# The FMP algorithm is included as a predefined `update_vel` function for use
+# with [`ContinuousSpace`](@ref).
+
+
+# ## Defining the Agent
+
+# First we need to define an `AbstractAgent` struct that has the necessary
+# information for the FMP algorithm to run. `pos`, and `vel` are position and
+# velocity, respectively. `tau` is an agents goal position. `color` is an
+# optional parameter for plotting used in this example. `type` is the agent
+# type: `:A` is an agent, `:O` is an obstacle, and `:T` is a target. `radius`
+# is the radius of the agent. `SSdims` is an optional parameter used for
+# plotting purposes.
+#
+# Agent types have the following distinctions:
+# - :A = agent (has velocity, subject to repulsive forces)
+# - :O = object (has no velocity, cannot move, agents must go around)
+# - :T = target (no velocity, just a projection of agent target zone)
+
+using Agents, Random, Plots, LinearAlgebra, Colors, ProgressMeter
+
+ENV["GKSwstype"]="nul"
+gr()
+cd(@__DIR__)
+
```

Comment:
Almost all of this is unecessary. Plotting now has moved to `InteractiveDynamics`. See for example the updated docs of the Flocking model, or the Social Distancing model. No reason for `ProgressMeter` either.